src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentMethodInfoPlugin.java

```diff
@@ -16,12 +16,12 @@
 
 package org.killbill.billing.plugin.adyen.api;
 
-import java.util.UUID;
```

Comment:
not sure what the goal is. Normally imports are added/removed and sorted by the IDE. With checked in code style (as discussed in the chat) there should be no change if the code was properly formatted (which is sadly not the case here).

I could remove the change but the next person will experience the same problem. An alternative would be to do a mass format as explicit commit. 

It's no fun to contribute if you always have to fear that pressing "format all" will kill every diff :(